### PR TITLE
Fixed privileges issues needed for coverage 

### PR DESCRIPTION
**Pull request by jgebal**
Added grant select any dictionary.

Fixed issue with weak password for ut3 user (some databases have password policies that require stronger passwords).
Added grant on DBMS_LOB, some user reported that this was not granted to public on his database.
Updated documentation to reflect changes.

### CODE REVIEW

File: .travis/install.sh [hunk 1]
```diff
@@ -9,8 +9,6 @@ set feedback off
 set verify off
 
 @../source/create_utplsql_owner.sql $UT3_OWNER $UT3_OWNER_PASSWORD $UT3_TABLESPACE
---only needed to run unit tests for utplsql v3, not required to run utplsql v3 itself
-grant select any dictionary to $UT3_OWNER;
```
Comment (Pazus) [inline .travis/install.sh]: lets try granting `SELECT_CATALOG_ROLE` role instead. I think it'll be enough

File: docs/userguide/install.md [hunk 2]
```diff
@@ -16,20 +16,26 @@ It is recommended to install utPLSQL in it's own schema. You are free to choose
 
 The installation user/schema must have the following Oracle system permissions during the installation.
   - CREATE SESSION
+  - CREATE SEQUENCE
   - CREATE PROCEDURE
   - CREATE TYPE
   - CREATE TABLE
   - CREATE VIEW
   - CREATE SYNONYM
+  - SELECT ANY DICTIONARY
```
Comment (Pazus) [inline docs/userguide/install.md]: Its too strong... i think `SELECT_CATALOG_ROLE` is enough

File: source/create_utplsql_owner.sql [hunk 3]
```diff
@@ -27,9 +27,10 @@ define ut3_tablespace = &3
 
 create user &ut3_user identified by &ut3_password default tablespace &ut3_tablespace quota unlimited on &ut3_tablespace;
 
-grant create session, create sequence, create procedure, create type, create table, create view, create synonym to &ut3_user;
+grant create session, create sequence, create procedure, create type, create table, create view, create synonym, select any dictionary to &ut3_user;
```
Comment (Pazus) [inline source/create_utplsql_owner.sql]: As this is only used by coverage reporter I dont think we should have it here. There will be lots of DBAs who wount give this grant even its on test server.

Comment (jgebal) [inline source/create_utplsql_owner.sql]: Aren't we're trying to satisfy DBA's needs too much?
I fear that If we will try to satisfy everyone's needs we will not progress and keep on stumbling upon issues.
We would probably need to separate coverage reporting capability from the main framework and make it a separate add-on but I don't really like the idea.

**How should we separate this privilege in your opinion?**

We could change the privilege to be one of:
- `select catalog role`
- `select on dba_source` (preffered)
- `select on all_source` (not so good, but used currently)
- `select_any_dictionary` (not tested)

Comment (Shoelace) [inline source/create_utplsql_owner.sql]: or not grant the privilege at all.. and use `AUTHID CURRENT_USER` and required that the user running the test has the correct permissions .

**Comment by jgebal** [comment]
I've found a different solution, so moving away from this one.

**Pull request closed without merging**